module/skill/QuinticWalk/src/QuinticWalk.hpp

```diff
@@ -0,0 +1,73 @@
+#ifndef MODULE_SKILL_QUINTICWALK_HPP
+#define MODULE_SKILL_QUINTICWALK_HPP
+
+#include <map>
+#include <memory>
+#include <nuclear>
+#include <vector>
+
+#include "extension/Behaviour.hpp"
+#include "extension/Configuration.hpp"
+
+#include "message/actuation/KinematicsModel.hpp"
+#include "message/behaviour/ServoCommand.hpp"
+#include "message/behaviour/state/Stability.hpp"
+
+#include "utility/input/ServoID.hpp"
+#include "utility/motion/WalkEngine.hpp"
+
+namespace module::skill {
+
+    class QuinticWalk : public ::extension::behaviour::BehaviourReactor {
+
+    public:
+        /// @brief Called by the powerplant to build and setup the QuinticWalk reactor.
+        explicit QuinticWalk(std::unique_ptr<NUClear::Environment> environment);
+
+        static constexpr int UPDATE_FREQUENCY = 200;
+
+    private:
+        // Reaction handle for the main update loop, disabling when not moving will save unnecessary CPU
+        // ReactionHandle update_handle{};
```

Comment:
Remove the update_handle line and update the comment to match that we only have it for the imu_reaction.